packages/xo-web/src/xo-app/xoa/update/index.js

```diff
@@ -275,328 +275,342 @@ const Updates = decorate([
     xoaTrialState,
     xoaUpdaterLog,
     xoaUpdaterState,
-  }) => (
-    <Container>
-      {COMMUNITY && (
-        <Row className='mb-1'>
-          <Col>
-            <p className='text-info'>{_('updaterCommunity')}</p>
-            <TryXoa page='updater' />
-          </Col>
-        </Row>
-      )}
-      <Row>
-        <Col mediumSize={6}>
-          <Card>
-            <CardHeader>
-              <UpdateTag /> {LABELS_BY_STATE[xoaUpdaterState]}
-            </CardHeader>
-            <CardBlock>
-              <fieldset disabled={COMMUNITY}>
-                <p>
-                  {_('xoaBuild')} {state.xoaBuild} - {_('currentVersion')}{' '}
-                  {defined(
-                    () => state.installedPackages['xen-orchestra'],
-                    'unknown'
-                  )}{' '}
-                  {state.installedPackages !== undefined && (
-                    <Button
-                      name='showPackagesList'
-                      onClick={effects.toggleState}
-                      size='small'
-                    >
-                      <Icon icon={state.showPackagesList ? 'minus' : 'plus'} />
-                    </Button>
-                  )}
-                </p>
-                {state.showPackagesList && (
-                  <p>
-                    <Copiable tagName='pre'>{state.packagesList}</Copiable>
-                  </p>
-                )}
-                {state.isDisconnected && (
+  }) => {
+    const xoVersion = get(() => state.installedPackages['xen-orchestra'])
+    const xoaBuild = state.xoaBuild
```

Comment:
`state.xoaBuild` is never `undefined?

Reply:
It could be `undefined` for sources users (`getApplianceInfo()` rejected). I changed it to ignore errors on the `getApplianceInfo` call. Now `xoaBuild` is `undefined` only during the first render and then quickly falls back to `'unknown'` for sources users.